scripts/adapters-container.dockerfile

```
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Build the test and build container for presto_cpp
+#
+FROM quay.io/centos/centos:stream8
+ARG cpu_target
+COPY scripts/setup-circleci.sh /
+COPY scripts/setup-helper-functions.sh /
+RUN mkdir build && ( cd build && CPU_TARGET="$cpu_target" bash /setup-circleci.sh ) && rm -rf build
```

Why build everything twice? ^^
```suggestion
FROM ghcr.io/facebookincubator/velox-dev:circleci-avx
```

We would also want to add this to the docker.yml workflow to automatically update it but probably as a second step with `needs:` to the matrix build so it uses the new image... not optimal but eh...